html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Billing.pm

```diff
@@ -0,0 +1,74 @@
+package pfappserver::Form::Config::Authentication::Source::Billing;
+
+=head1 NAME
+
+pfappserver::Form::Config::Authentication::Source::Billing;
+
+=cut
+
+=head1 DESCRIPTION
+
+pfappserver::Form::Authentication::Source::Billing
+
+=cut
+
+use strict;
+use warnings;
+use HTML::FormHandler::Moose;
+extends 'pfappserver::Form::Config::Authentication::Source';
+
+has_field currency => (
```

Comment:
Shouldn't that be a select or point to a link where the valid currencies are displayed ?